blog/_posts/2022-11-20-gsoc-report.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: blog-detail
+post-type: blog
+by: Anatolii Kmetiuk, Scala Center
+title: "Google Summer of Code 2022: How It Was"
+---
+We've recently completed another successful Google Summer of Code program. Designed to bring more beginners into programming communities, the program is a part of Scala Center's strategy to make the Scala language more newcomer-friendly.
```

Comment:
I don’t think the goal is to make Scala more newcomer-friendly with GSoC. I would rather say that the goal is to onboard new contributors to the Scala ecosystem.

Reply:
I rather like this part, and it is correct that our participation in GSoC is a part/ one of the pieces of our strategy to make Scala lang more newcomer-friendly.